trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -1328,6 +1371,7 @@ impl Display for InstructionAndBytesParsedFromBuf {
 }
 
 // TODO: Also move behind a feature flag...
+#[allow(dead_code)]
 fn parse_to_instructions_and_bytes_for_instruction(
```

Comment:
Do we plan on using it? Otherwise I'd just go for removing it

Reply:
Same as above.